lib/fluent/plugin/output.rb

```diff
@@ -185,13 +185,26 @@ def initialize
         @simple_chunking = nil
         @chunk_keys = @chunk_key_time = @chunk_key_tag = nil
         @flush_mode = nil
+        @timekey_zone = nil
       end
 
       def acts_as_secondary(primary)
         @as_secondary = true
         @primary_instance = primary
+
+        @chunk_keys = @primary_instance.chunk_keys if @primary_instance.chunk_keys
```

Comment:
I changed this method name to `empty?`

https://github.com/fluent/fluentd/pull/1154/commits/3a341875c87c5c1bc110b70abf1bddddff0895a9